packages/customer-account-ui-extensions-react/src/hooks/authenticated-redirect.ts

```diff
@@ -0,0 +1,13 @@
+import {
+  RenderExtensionPoint,
+  StandardApi,
+} from '@shopify/customer-account-ui-extensions';
+
+import {useApi} from './api';
+
+// Returns authenticated redirect api utilities for the extension point.
+export function useAuthenticatedRedirect<
```

Comment:
that's on me since we didn't remove it yet, but `authenticatedRedirects` are dead and we should not expose a hook. Can you please remove it?